### ISSUE TITLE
Feature request: native midi Kotlin example

### ISSUE THREAD

**Issue by sensn** [android/ndk-samples#963]
It would be great to have a Kotlin version of the native_midi example app.

**Comment by DanAlbert** [comment]
Why? If you're looking for better examples of how to do Kotlin UIs, the NDK samples really aren't the place for that.

If I've misunderstood something lmk, but I don't think it's feasible (or even helpful) for us to have a Kotlin and Java variant of NDK samples.

**Comment by sensn** [comment]
Hello,

thanks for your reply. It's not so much about the UI, but how the NDK Midi
library is linked and initialized from the Java/Kotlin side.
When I opened the issue I was trying to migrate the Java example to Kotlin,
but I ran into a lot of issues, and couldn't resolve an error about how to
init the nativelib, although I used a companion object.
Furthermore I read that there were some improvements to AMidi API and was
wondering if they are reflected in the samples.
I think docs have become more detailed in the meantime, so I might give it
another try.
Thanks.

Dan Albert ***@***.***> schrieb am Fr., 3. Mai 2024, 23:31:
